Seminar5/Task2.py

```diff
@@ -0,0 +1,10 @@
+# Напишите функцию, которая принимает на вход строку - абсолютный путь до файла. 
+# Функция возвращает кортеж из трёх элементов: путь, имя файла, расширение файла.
+
+link = "Seminar5//txt.txt".replace(".", "//")
```

Comment:
Если добавить это, то программа завершается с ошибкой.
```suggestion
link = "Home//Seminar5//txt.txt".replace(".", "//")
```